src/shared/components/StickySaveBar/StickySaveBar.scss

```diff
@@ -5,6 +5,7 @@
 	flex-direction: column;
 	flex-grow: 1;
 	justify-content: space-between; // Keep the save indicator at the bottom
+	min-height: calc(100vh - 40rem); // Ensure save bar stays at the bottom when not enough content
```

Comment:
These lines do the same thing; are all the parent containers around `.c-container--sticky-save-bar-wrapper` filling their own parents so `space-between` can do its thing?

Reply:
The min-height seems to be needed to make the container large enough when not enough content is on the page

This is what i think is happening:
![image](https://user-images.githubusercontent.com/1710840/177993790-b82f089a-a131-406b-8791-7ebe9136bddc.png)